src/sentry/incidents/models.py

```diff
@@ -524,51 +525,29 @@ class Meta:
 
 
 @region_silo_only_model
-class AlertRuleTriggerAction(Model):
+class AlertRuleTriggerAction(AbstractNotificationAction):
     """
     This model represents an action that occurs when a trigger is fired. This is
     typically some sort of notification.
     """
 
+    # Aliases from NotificationAction
+    Type = ActionService
+    TargetType = ActionTarget
+
     __include_in_export__ = True
```

Comment:
Nit: I'd probably just leave this at the very top of the class, just tends to be our convention